common/src/main/nl/jft/common/Main.java

```diff
@@ -0,0 +1,37 @@
+package nl.jft.common;
+
+import nl.jft.common.util.elo.EloExpectationCalculator;
+import nl.jft.common.util.elo.EloExpectationResult;
+import nl.jft.common.util.elo.loader.FileEloExpectationLoader;
+import nl.jft.common.util.elo.strategy.LinearAlphaStrategy;
+
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author Oscar de Leeuw
+ */
+public class Main {
+    private static final Logger LOGGER = Logger.getLogger(Main.class.getName());
+
+    public static void main(String[] args) {
+        //InputStream is = Main.class.getResourceAsStream("resources/estimates.txt");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)